test/gob/gob.go

```diff
@@ -7,10 +7,36 @@ import (
 	"log"
 	"os"
 	"os/signal"
-
-	"github.com/aquasecurity/tracee/tracee"
+	"time"
 )
 
+// Event is a user facing data structure representing a single event
+type Event struct {
```

Comment:
Why having a copy of the struct here?

Reply:
Same reason we have a copy in tracee-rule. Importing entire tracee just for the struct is too much. Also tracee requires linux to compile which is not a requirement for this program. In the future I plan to move these types outside the tracee package and then we could import them